src/plugins/console/server/lib/spec_definitions/json/generated/connector.sync_job_list.json

```diff
@@ -0,0 +1,38 @@
+{
+  "connector.sync_job_list": {
+    "url_params": {
+      "error_trace": "__flag__",
+      "filter_path": [],
+      "human": "__flag__",
+      "pretty": "__flag__",
+      "from": "",
+      "size": "",
+      "status": [
+        "canceling",
+        "canceled",
+        "completed",
+        "error",
+        "in_progress",
+        "pending",
+        "suspended"
+      ],
+      "connector_id": "",
+      "job_type": [
+        "full",
+        "incremental",
+        "access_control"
+      ]
+    },
+    "methods": [
+      "GET"
+    ],
+    "patterns": [
+      "_connector/_sync_job"
```

Comment:
For this endpoint, many of the URL params are not auto-suggested and the documentation link is incorrect. Probably the autocomplete functionality confuses this endpoint with `GET _connector/_sync_job/{connector_sync_job_id}`? 🤔

Reply:
Nice catch @ElenaStoeva! Indeed, the autocomplete engine uses the endpoint `connector.get` because the pattern is `"_connector/{connector_id}"` so `connector/_sync_job` is matched there. I'll open an issue to fix this in the autocomplete code.